composer.json

```diff
@@ -1,31 +1,33 @@
 {
     "name": "madmatt/silverstripe-encrypt-at-rest",
+    "type": "silverstripe-vendormodule",
     "description": "Enable encryption of data at rest (in database)",
-    "version": "0.0.1",
-    "type": "silverstripe-module",
     "keywords": [ "silverstripe", "encryption", "decryption", "encrypt-at-rest" ],
     "homepage": "https://github.com/madmatt/silverstripe-encrypt-at-rest",
     "license": "WTFPL",
-
     "authors": [
         {
             "name": "Matt Peel",
             "email": "matt@silverstripe.com"
         }
     ],
-
     "support": {
         "issues": "https://github.com/madmatt/silverstripe-encrypt-at-rest/issues",
         "source": "https://github.com/madmatt/silverstripe-encrypt-at-rest"
     },
-
     "require": {
-        "silverstripe/framework": "^3.7",
-        "defuse/php-encryption": "v2.2.1"
+        "silverstripe/framework": "^4.1",
```

Comment:
This needs to be `^4.9.0` now as that's when the `framework` PR was merged.

Reply:
Cheers @madmatt, have updated